frontend/src/Onboarding/OnboardingInfoScreen.tsx

```diff
@@ -356,9 +400,20 @@ class OnboardingScreenComponent extends React.Component<
           {this.renderCatalogYearDropDown()}
           {/* if there is a major given from khoury we want to show the major dropdown */}
           {(!!catalogYear || !!major) && this.renderMajorDropDown()}
+          {!!catalogYear && !!major && this.renderConcentrationDropdown()}
```

Comment:
Do we also want to check if concentrations exist for this major?

Reply:
We handle that case within the concentration dropdown component itself [here](https://github.com/sandboxnu/graduatenu/pull/284/files#diff-69a90420eca89202b239cf9399edc60fa6adbdd4d819c8c2dcd9b9cfc7d4cf25R49-R50)